omod_1.x/src/main/java/org/openmrs/module/openhmis/cashier/web/controller/CashierMessageRenderController.java

```diff
@@ -0,0 +1,27 @@
+package org.openmrs.module.openhmis.cashier.web.controller;
+
+import org.openmrs.module.openhmis.backboneforms.web.BackboneWebConstants;
+import org.openmrs.module.openhmis.cashier.web.CashierWebConstants;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.support.RequestContextUtils;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+/**
```

Comment:
Remove this comment. This information is stored in the VCS and isn't needed here.